pkg/auth/asynccache.go

```diff
@@ -10,6 +10,11 @@ import (
 	"k8s.io/klog"
 )
 
+const (
+	initializationRetries    = 10
+	initializationRetryDelay = 30 * time.Second
```

Comment:
you wan to be under the 150 liveness probe delay, which I think means you want this delay to 11 seconds.

Reply:
See https://github.com/openshift/console-operator/pull/869 which changes the probes for the console container.